src/plugins/flow.js

```diff
@@ -252,21 +258,24 @@ export default (superClass) => class extends superClass {
         node = this.parseExport(node);
         if (node.type === "ExportNamedDeclaration") {
           // flow does not support the ExportNamedDeclaration
+          // $FlowIgnore
           node.type = "ExportDeclaration";
+          // $FlowFixMe
           node.default = false;
           delete node.exportKind;
         }
 
+        // $FlowIgnore
         node.type = "Declare" + node.type;
```

Comment:
Type systems don't handle objects changing type very well. There are similar problems in the estree plugin.